packages/test/test-end-to-end-tests/src/test/gc/gcTombstoneAttachmentBlobs.spec.ts

```diff
@@ -268,6 +268,80 @@ describeNoCompat("GC attachment blob tombstone tests", (getTestObjectProvider) =
             assert.strictEqual(response?.status, 200, `Expecting a 200 response`);
             assert(container2.closed !== true, "Container should not have closed");
         });
+
+        /**
+         * Function that inverses the does not reject condition, i.e., asserts for reject instead. Used in these tests
+         * to demonstrate that because of a bug we are not getting the expected results. Once the bug is fixed, these
+         * asserts should start working as expected.
+         */
+        async function assertDoesNotRejectInverse(block: Promise<any>, message?: string | Error) {
```

Comment:
nit: Maybe `assertWronglyRejects`? Or just use `assert.rejects` directly and put a comment like `// BUG BUG BUG` haha

Reply:
assertWronglyRejects is better.